src/charts/donut/donut-pct-chart.html

```diff
@@ -1,3 +1,4 @@
 <span>
-  <div pf-c3-chart id={{donutChartId}} config="config"></div>
+  <div pf-c3-chart ng-if="data.dataAvailable !== false" id={{donutChartId}} config="config"></div>
+  <div pf-empty-chart ng-if="data.dataAvailable === false" chart-height="172"></div>
```

Comment:
172 will work for the default donut size (set in $().c3ChartDefaults().getDefaultDonutConfig()), but if the caller specified a different size, this size will not work. This should either be a parameter to the directive, or computed based of the donut chart height.